integration-tests/tests/preview-drs-uri.js

```diff
@@ -33,6 +34,7 @@ const testPreviewDrsUriFn = _.flow(
   await click(page, clickable({ textContains: 'View Workspaces' }));
   await waitForNoSpinners(page);
   await fillIn(page, input({ placeholder: 'Search by keyword' }), workspaceName);
+  await findElement(page, clickable({ textContains: workspaceName }));
```

Comment:
I thought `await click()` would do this automatically. If it doesn't shouldn't we instead make a wrapper that does, instead of needed to call it before calling `click()`? When would we want to try to click if the element is not yet visible?

Regarding verifying that a change actually improves flakiness, I would use the flake shaker to do that. If flakiness is improved, the output of running with/without a change should show that.

Reply:
I see, I will try it! Previous attempts to use test-flakes weren't working for me but i will have another go.